src/components/library/library.css

```diff
@@ -10,7 +10,7 @@
     flex-grow: 1;
     flex-wrap: wrap;
     overflow-y: auto;
-    height: calc(100% - $library-header-height);
+    height: auto;
```

Comment:
I checked all libraries: costume/library/tips/extensions and they all seem ok to me on Chrome/MacOS with this fix.